servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -245,17 +233,33 @@ def _handle_http_error(error, proj_id):
 def _metadata_response(id, job):
     return JobMetadataResponse(
         id=id,
-        name=job['job-name'],
         status=job_statuses.dsub_to_api(job),
         submission=job['create-time'],
+        name=job['job-name'],
         start=job.get('start-time'),
         end=job['end-time'],
         inputs=job['inputs'],
         outputs=job['outputs'],
-        labels=labels.dsub_to_api(job),
-        logs=logs.dsub_to_api(job),
-        failures=failures.get_failures(job))
+        labels=labels.get_labels(job),
+        failures=failures.get_failures(job),
+        extensions=extensions.get_extensions(job))
 
 
 def _provider_type():
     return current_app.config['PROVIDER_TYPE']
+
+
+def _query_jobs_result(job, project_id=None):
+    return QueryJobsResult(
+        id=job_ids.dsub_to_api(project_id, job['job-id'], job.get(
+            'task-id', '')),
```

Comment:
nit: I'd probably make it accept `None` for task-id

Reply:
Yup, already does not sure why it was like this.